src/tracing/CMakeLists.txt

```diff
@@ -45,8 +47,10 @@ set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
 add_tracing_library(rados_tp librados.tp 2.0.0)
 add_tracing_library(os_tp objectstore.tp 1.0.0)
+add_tracing_library(rgw_op_tp rgw_op.tp 1.0.0)
+add_tracing_library(rgw_rados_tp rgw_rados.tp 1.0.0)
```

Comment:
there's a build failure in radosgw:
> rgw_op.cc:55:10: fatal error: tracing/rgw_op.h: No such file or directory

it looks like radosgw will need to add dependencies to make sure that the headers get generated first. something like this in `src/rgw/CMakeLists.txt`:
```cmake
if(WITH_LTTNG)
  add_dependencies(rgw_a rgw_op-tp rgw_rados-tp)
endif()
```

Reply:
@cbodley Yes, I forgot to add this dependence.